dataservice/api/biospecimen/models.py

```diff
@@ -0,0 +1,78 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base, KfId
+from dataservice.api.genomic_file.models import GenomicFile
+
+
+class Biospecimen(db.Model, Base):
+    """
+    Biospecimen entity.
+    :param _id: Unique id assigned by RDBMS
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param external_sample_id: Name given to sample by contributor
+    :param external_aliquot_id: Name given to aliquot by contributor
+    :param composition : The cellular composition of the biospecimen.
+    :param tissue_type: description of the kind of tissue collected
+           with respect to disease status or proximity to tumor tissue
+    :param anatomical_site : The name of the primary disease site of the
+           submitted tumor biospecimen
+    :param age_at_event_days: Age at the time biospecimen was
+            acquired, expressed in number of days since birth
+    :param tumor_descriptor: The kind of disease present in the tumor
+           specimen as related to a specific timepoint
+    :param shipment_origin : The origin of the shipment
+    :param shipment_destination: The destination of the shipment
+    :param analyte_type: Text term that represents the kind of molecular
+           specimen analyte
+    :param concentration: The concentration of an analyte or aliquot extracted
+     from the biospecimen or biospecimen portion, measured in
+     milligrams per milliliter
+    :param volume: The volume in microliters (ml) of the aliquots derived from
+     the analyte(s) shipped for sequencing and characterization
+    :param shipment_date: The date item was shipped in YYYY-MM-DD format
+    :param uberon_id: The ID of the term from Uber-anatomy ontology
+     which represents harmonized anatomical ontologies
+    """
+    __tablename__ = 'biospecimen'
+    __prefix__ = 'BS'
+
+    external_sample_id = db.Column(db.Text(),
+                                   doc='Name given to sample by contributor')
+    external_aliquot_id = db.Column(db.Text(),
+                                    doc='Name given to aliquot by contributor')
+    tissue_type = db.Column(db.Text(),
+                            doc='Description of the kind of biospecimen '
+                            'collected')
+    composition = db.Column(db.Text(),
+                            doc='The cellular composition of the biospecimen')
+    anatomical_site = db.Column(db.Text(),
```

Comment:
Shouldn't this be covered by `uberon_id`?

Reply:
it's storing raw anatomical_site

Reply:
This is the raw text field from which uberon_id is derived, right?

Reply:
What would we name it? `raw_anatomical_site`?

Reply:
ya. we can rename other ontology's raw texts too in this standard. for suppose adding `raw_` to original field name.

Reply:
How about `source_`? and yeah would be nice to have this consistent across the tables.

Reply:
How about `source_`? and yeah would be nice to have this consistent across the tables.

Reply:
We already use `raw_text` for diagnosis+phenotype. Maybe `raw_` is better

Reply:
Created a separate issue for changing the ontology raw fields #237